scalariform/src/test/scala/scalariform/formatter/CommentFormatterTest.scala

```diff
@@ -99,6 +99,44 @@ class CommentFormatterTest extends AbstractFormatterTest {
     | */
     |"""
 
+  """/**
+    | * Trailing whitespace on this line and the line below should be stripped.   
+    | *    
```

Comment:
We don't want to have trailing whitespace in our code. Please append a `$` sign at the end (or some other character) and strip it away afterwards with `str.replace("$", "")`